motoman_gp200r_support/urdf/gp200r_macro.xacro

```diff
@@ -0,0 +1,164 @@
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_2_l.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_2_l.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_3_u">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_3_u.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_3_u.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_4_r">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_4_r.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_4_r.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_5_b">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_5_b.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_5_b.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_6_t">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_6_t.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_6_t.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}tool0"/>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint name="${prefix}joint_1_s" type="revolute">
+        <parent link="${prefix}base_link"/>
+        <child link="${prefix}link_1_s"/>
+        <origin xyz="0 0 0.450" rpy="0 0 0" />
+        <axis xyz="0 0 1" />
+        <limit lower="${radians(-180)}" upper="${radians(180)}" effort="9414.38" velocity="${radians(125)}"/>
```

Comment:
Could you clarify where you got the `effort` values from?

Reply:
I usually use the Max Acc/Dec Torque as the effort values that I extract from the robot parameters:
Please refer to this file:
[Robot_Specs_GP200R-A00.csv](https://github.com/ros-industrial/motoman/files/7990153/Robot_Specs_GP200R-A00.csv)
:

Reply:
I used the effort values in the [Robot_Specs_GP200R-A00.csv](https://github.com/ros-industrial/motoman/files/7990153/Robot_Specs_GP200R-A00.csv) doc that Eric provided. Thank you!